src/main/webapp/app/view/project/admin/locations/project-location-update.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpResponse } from '@angular/common/http';
+import { FormBuilder, Validators } from '@angular/forms';
+import { Observable } from 'rxjs';
+
+import { Project } from 'app/shared/model/project.model';
+import { Location } from 'app/shared/model/location.model';
+
+import { LocationService } from 'app/entities/location/location.service';
+import { Responsibility } from 'app/shared/model/responsibility.model';
+import { ResponsibilityService } from 'app/entities/responsibility/responsibility.service';
+
+@Component({
+  selector: 'app-location-update',
+  templateUrl: './project-location-update.component.html',
+  styleUrls: ['./project-location-update.component.scss'],
+})
+export class ProjectLocationUpdateComponent implements OnInit {
+  isSaving = false;
+  isNew = false;
+  project?: Project;
+  responsibilities: Responsibility[] = [];
+  //filteredResponsibilities?: Observable<Responsibility[]>;
```

Comment:
du musst `filteredResponsibilities` drin lassen, kannst das aber auch als `: Responsibility[] = []` deklarieren. als `options` auf den autocomplete kommen dann nur diese. die `responsibilities` selbst sind `private` und werden niemals verändert, abgesehen vom `ngOnInit`.
die interne `filter` methode, verändert dann nur die `filteredResponsibilities` und hat den Rückgabewert `void`. es ist ein vom autocomplete aufgerufenes "callback".

in der `User` Umsetzung (#89) habe ich zwar nur ein Array `filteredUsers: User[] = [];`, in der Methode `filterUsersByLoginOrEmail(value: string): void` rufe ich jedoch immer den Service auf. Dies, da ich die User nicht nach Projekt filtern kann und das potentiel sehr viele Daten sind.